components/file_preview_modal/file_preview_modal_image_controls/file_preview_modal_image_controls.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {ChangeEvent, memo, useEffect, useState} from 'react';
+import {PlusIcon, MinusIcon} from '@mattermost/compass-icons/components';
+import {useIntl} from 'react-intl';
+import {clamp} from 'lodash';
+
+import {minZoom, zoom} from '../image_preview';
+import './file_preview_modal_image_controls.scss';
+
+let zoomInButtonDisabled = false;
+let zoomOutButtonDisabled = true;
```

Comment:
Sry I did not catch this earlier: It is generally not a good idea to have variables declared outside of a component. This is fine for constants, but whenever you are mutating the values it is better to use a `ref`